unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/Tabs/FriendsTabView.cs

```diff
@@ -137,4 +145,31 @@ private void ChatController_OnAddMessage(ChatMessage message)
 
         lastProcessedFriend = friend.userId;
     }
+
+    public void RequestCreateEntry(string userId, FriendEntryBase.Model model)
+    {
+        creationQueue[userId] = model;
+    }
+
+    protected override void ProcessInFrame()
```

Comment:
I'd add some commenting here as well to at least explain how this works and the edge cases addressed.